tools/please_pex/pex_main.py

```diff
@@ -1,12 +1,12 @@
 """Zipfile entry point which supports auto-extracting itself based on zip-safety."""
 
-import fcntl
 from importlib import import_module
 from zipfile import ZipFile, ZipInfo, is_zipfile
 import os
 import runpy
 import sys
 
+
```

Comment:
Is this formatting change desired?

Reply:
Yeah PEP8 says two blank lines after imports. Not that it deeply matters much :)